mappings/net/minecraft/SignedChatMessageC2SPacket.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_7438 net/minecraft/SignedChatMessageC2SPacket
```

Comment:
This class is S2C, not C2S. Also, it should not be in top-level `net.minecraft` package.